chromium/background.js

```diff
@@ -370,18 +370,17 @@ function onBeforeRequest(details) {
     }
   }
 
-  if (using_credentials_in_url) {
+  if (newuristr) {
```

Comment:
This isn't quite what I meant.  Sill use the `if (using_credentials_in_url) {` but inside it add the conditional `if (newuristr) {`, with an `} else {` clause.